deploy/charts/templates/_helpers.tpl

```diff
@@ -106,14 +111,14 @@ Create match labels for ndm operator deployment
 {{- define "openebs-ndm.operator.matchLabels" -}}
 app: {{ template "openebs-ndm.operator.name" . }}
 release: {{ .Release.Name }}
-component: {{ .Values.ndmOperator.name | quote }}
+component: {{ default (include "openebs-ndm.operator.name" .) .Values.ndmOperator.componentName }}
 {{- end -}}
 
 {{/*
 Create component labels for ndm operator component
 */}}
 {{- define "openebs-ndm.operator.componentLabels" -}}
-openebs.io/component-name: {{ .Values.ndmOperator.name | quote }}
+openbes.io/component-name: {{ default (include "openebs-ndm.operator.name" .) .Values.ndmOperator.componentName }}
```

Comment:
```suggestion
openebs.io/component-name: {{ default (include "openebs-ndm.operator.name" .) .Values.ndmOperator.componentName }}
```

Reply:
That has to be in a single line.

Reply:
Oops. Sorry this is not about singe line / multi line. There was a typo in `openebs`

Reply:
Oh. Ok thank you. I missed that.